TutorialApplication/MainViewController.m

```diff
@@ -101,4 +125,34 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
     }
 }
 
+- (BOOL)tableView:(UITableView *)tableView canEditRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    return YES;
+}
+
+- (void)tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEditingStyle)editingStyle forRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    NSManagedObjectContext *context = [DatabaseController sharedInstance].context;
+    
+        NSMutableArray *arrayEmployees = [NSMutableArray new];
+        arrayEmployees = [self.myEmployees mutableCopy];
+    
+    if (editingStyle == UITableViewCellEditingStyleDelete)
+    {
+        // Delete object from database
+        [context deleteObject:[arrayEmployees objectAtIndex:indexPath.row]];
```

Comment:
Add a method to organization class which removes employee object. Use CoreData's specific functionality shouldn't be in a controller.